workflow/automatic-workflow/workflowtemplates/110-tft.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: WorkflowTemplate
+metadata:
+  name: data-transform
+  labels:
+    tensorstack.dev/component: taskman
```

Comment:
pls ensure these labels are expected and consistent.

Reply:
ditto